source/proof_document/lib/proof_document.rb

```diff
@@ -0,0 +1,149 @@
+require 'bundler/setup' if !defined?(Bundler)
+require 'faraday'
+require 'json'
+require 'retries'
+require '/opt/ruby/lib/faraday_helper' if !defined?(IdentityIdpFunctions::FaradayHelper)
+require '/opt/ruby/lib/ssm_helper' if !defined?(IdentityIdpFunctions::SsmHelper)
+require 'openssl'
+require "aws-sdk-s3"
```

Comment:
can we rebase this from `master` branch? it'll include the rubocop fixes and catch this string quoting style